src/com/hrabrov/introduction_to_java/algorithmization/multidimensional_array/StringHelper.java

```diff
@@ -0,0 +1,44 @@
+package com.hrabrov.introduction_to_java.algorithmization.multidimensional_array;
+
+public class StringHelper {
+    public static String matrixToString(int[][] matrix) {
+        String stringForStorageOfMatrixValues = "";
+
+        for (int i = 0; i < matrix.length; i++) {
+            for (int j = 0; j < matrix[i].length; j++) {
+                stringForStorageOfMatrixValues += matrix[i][j] >= 0 ? " " : "";
+                stringForStorageOfMatrixValues += matrix[i][j];
+                stringForStorageOfMatrixValues += matrix[i][j] < 100 ? "\t" : "";
```

Comment:
There should be no numbers in the code, all such values should be declared as constants and have meaningful name